### PR TITLE
Wemos helper file implementation

### PR DESCRIPTION
**Pull request by HowManyOliversAreThere**
Added a basic Wemos D1 Mini pin mapping file, along with the CI integration to lint and check formatting of (Micro)Python files in the repo.

I would've preferred pin mapping to be done through an enum or similar, to retain the `.` syntax but be encapsulated instead of just globals in the file, but given the options I believe this gave the best user experience.

Usage:
```python
import wemos, machine, time
led = machine.Pin(wemos.LED, machine.Pin.OUT)
while True:
    led.value(not led.value())
    time.sleep(1)
```

### CODE REVIEW

File: python/wemos.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+""" Helper file for the Wemos D1 Mini
+    https://wiki.wemos.cc/products:d1:d1_mini
+"""
+
+# Pins
```
Comment (mattytrentini) [inline python/wemos.py:5]: Should probably make these const...

File: .github/workflows/pythonpackage.yml [hunk 2]
```diff
@@ -30,3 +30,11 @@ jobs:
     - name: Make html pages
       run: |
         make html
+    - name: Lint Python files
+      run: |
+        pip install pylint
+        pylint python
+    - name: Check Python file formatting
+      run: |
+        pip install black
```
Comment (mattytrentini) [inline .github/workflows/pythonpackage.yml:39]: Love it. :) Could possibly make it a pre-commit hook...

File: .github/workflows/pythonpackage.yml [hunk 3]
```diff
@@ -30,3 +30,11 @@ jobs:
     - name: Make html pages
       run: |
         make html
+    - name: Lint Python files
+      run: |
+        pip install pylint
+        pylint python
```
Comment (mattytrentini) [inline .github/workflows/pythonpackage.yml:36]: pylint is great, flake8 maybe even better

**Comment by HowManyOliversAreThere** [comment]
Added basic instructions to Getting Started page, that utilises this helper function implementation

**Review by mattytrentini — COMMENTED** [review]
Looks good! Quite descriptive - and wordy - but that's probably good to start with.

**Pull request merged**